controllers/volsync/vshandler.go

```diff
@@ -578,21 +562,21 @@ func (v *VSHandler) pvcExistsAndInUse(pvcName string, inUsePodMustBeReady bool)
 	return util.IsPVAttachedToNode(v.ctx, v.client, v.log, pvc)
 }
 
-func (v *VSHandler) pvcExists(pvcName string) (bool, error) {
-	_, err := v.getPVC(pvcName)
+func (v *VSHandler) pvcExists(pvcName string) (bool, *corev1.PersistentVolumeClaim, error) {
+	pvc, err := v.getPVC(pvcName)
 	if err != nil {
 		if !kerrors.IsNotFound(err) {
 			v.log.V(1).Info("failed to get PVC")
 
-			return false, err
+			return false, pvc, err
 		}
 
-		return false, nil
+		return false, pvc, nil
 	}
 
 	v.log.V(1).Info("PVC found")
 
-	return true, nil
+	return true, pvc, nil
```

Comment:
true, pvc, nil, can be changed to pvc, nil

Reply:
Ok